src/python_be.cc

```diff
@@ -1515,15 +1515,6 @@ ModelInstanceState::SendBLSDecoupledResponse(
     }
   }
 
-  ScopedDefer _([&ipc_message, response_batch] {
-    {
-      bi::scoped_lock<bi::interprocess_mutex> lock{
-          *(ipc_message->ResponseMutex())};
-      response_batch->waiting_on_stub = false;
```

Comment:
This line has changed waiting_on_stub to false first, then notify the pb_stub. But why pb_stub's waiting_on_stub is still true after being waken up. @krishung5